frontend/src/component/project/Project/ProjectDoraMetrics/ProjectDoraMetrics.tsx

```diff
@@ -90,7 +91,23 @@ export const ProjectDoraMetrics = () => {
                         {original.timeToProduction} days
                     </Box>
                 ),
-                width: 150,
+                width: 200,
+                disableGlobalFilter: true,
+                disableSortBy: true,
+            },
+            {
+                Header: `Deviation`,
+                id: 'Deviation from average',
+                align: 'center',
+                Cell: ({ row: { original } }: any) => (
+                    <Box
+                        sx={{ display: 'flex', justifyContent: 'center' }}
+                        data-loading
+                    >
+                        {dora.projectAverage - original.timeToProduction} days
```

Comment:
`dora` seems to be missing any types. It should come from hook.

Reply:
Backend needs to be merged before I can generate from enterprise